
 <launch>
<group ns = "robot_0">
  <node pkg="robots_deployment_online" type="deployment.py" name="ros_deployment_0" output="screen">
    <param name="pose_send_time" value="0.3" />
    <param name="id" value="0" />
    <param name="tree_file" value="/home/elerson/catkin_ws/src/networked_robots_experimetal_data/1euclideanexperiment/steinerData1.dat" />
    <param name="map_resolution" value="0.05" />
  </node> 


 <node pkg="fake_localization" type="fake_localization" name="amclrobot_0" output="screen">
   <!-- Publish scans from best pose at a max of 10 Hz -->
   <param name="odom_model_type" value="omni"/>
   <param name="base_frame_id" value="robot_0/base_link"/>
   <param name="odom_frame_id" value="robot_0/odom"/>
   <param name="global_frame_id" value="/map"/>
  </node>


</group>

 </launch>
